### ISSUE TITLE
"Uncaught ReferenceError : require is not defined" after installing with Bower

### ISSUE THREAD

**Issue by jordancrawfordnz** [jtblin/angular-chart.js#466]
### Overview

I setup the npm resolver then installed the latest version with Bower. My app was created with the Yeoman Angular generator, so my bower dependencies autowire.

I got the message "Uncaught ReferenceError : require is not defined" in the console window, coming from the chart.js file. Based on http://stackoverflow.com/questions/38176769/uncaught-referenceerror-require-is-not-defined-chart-js, I manually changed the import in my HTML to go to the /dist/Chart.bundle.min.js file and this issue went away.

I applied a manual override in Bower like:
```
"overrides" : {
   "chart.js": {
      "main": [
        "dist/Chart.bundle.min.js"
      ]
    }
}
```

Has anyone else got this issue? Possibly a Bower override could be added in the bower.json to resolve this?

### Step to reproduce
As above.

**Comment by srijken** [comment]
I have the same issue. Override doesn't seem to work, possibly because of the bower npm resolver?

**Comment by TimmyParkwayd** [comment]
I have the same issue too

**Comment by jordancrawfordnz** [comment]
The override is only required if you're automatically populating the HTML imports from the Bower dependencies like I am. If you aren't, you should be able to just manually change the imported file to be the ``dist/Chart.bundle.min.js`` one rather than ``src/chart.js``.

**Comment by srijken** [comment]
I'm indeed generating script includes based on the bower dependencies.

Just realized I was trying to override the main in "exportOverrides", that didn't work. Put it in "overrides", and it works for me now. Thanks @jordancrawfordnz

**Comment by TimmyParkwayd** [comment]
I automatically populating the HTML imports from bower dependencies .
Here is my 2 errors 
<img width="1440" alt="capture d ecran 2016-08-17 a 17 06 06" src="https://cloud.githubusercontent.com/assets/1898326/17741569/fd91ddb8-649c-11e6-8a62-68a11d0366d0.png">

**Comment by jordancrawfordnz** [comment]
No worries @srijken.
@TimmyParkwayd, is the Chart.bundle.min.js file present at that location? Maybe you have an older version of Chart.js that doesn't have that file?

**Comment by TimmyParkwayd** [comment]
Yes it's at the right location . 
<img width="281" alt="capture d ecran 2016-08-18 a 10 13 55" src="https://cloud.githubusercontent.com/assets/1898326/17766672/90c157b4-652c-11e6-9b34-bc71ce73bca1.png">

I'm trying to use the 1.0.0 version now thanks to the issue 467.

**Comment by jtblin** [comment]
I'm happy to include the override in the `bower.json` file if that solves the issue. Can someone confirms that?

**Comment by TimmyParkwayd** [comment]
It could fix some of the issues yes :)

**Comment by TimmyParkwayd** [comment]
I also try to overide and user v1.0.1 of Chart.js but I now have this error

<img width="1434" alt="capture d ecran 2016-08-18 a 11 15 47" src="https://cloud.githubusercontent.com/assets/1898326/17768650/3f552fa0-6535-11e6-83ed-6fc6f8125214.png">

**Comment by jibusayone** [comment]
I'm facing the same issue. Please resolve.

**Comment by cedricCdl** [comment]
I find a solution.

bower.json :
"angular-chart.js": "^1.0.1",
"chart.js": "npm:chart.js#2.1.0"

it works !!

**Comment by qgliu** [comment]
@cedricCdl You solved my problem. I guess the issue is that by default, the chart 2.2.1 (as of August 18) is installed. This version is not recognizable by angular-chart.js

**Comment by jtblin** [comment]
They only removed support for bower in 2.2 I believe.

**Comment by jtblin** [comment]
Released `1.0.2` to address this issue.

**Comment by GabLeRoux** [comment]
@cedricCdl Your solution seemed to work, but it actually fails on windows;

    ls ./node_modules/
    ...
    Chart.js
    chart.js
    ...

Works on Case sensitive os, but not on others. That's annoying.

**Comment by jibusayone** [comment]
@cedricCdl Thanks a lot mate. Worked like a charm!

**Comment by ekaram** [comment]
Still having an issue with the above solution: 
     ..
    "angular-chart.js": "^1.0.1",
    "chart.js": "npm:chart.js#2.1.0"
  },
"resolvers": [
    "bower-npm-resolver"
  ]
}
> bower install
... 
bower chart.js#2.1.0                                               ENOTFOUND Package npm:chart.js not found

**Comment by qgliu** [comment]
@ekaram Try change the angular-chart to fixed 1.0.1 by removing ^? You might picked up the upgraded angular-chart (1.0.2). I am not sure though.

**Comment by ekaram** [comment]
Thanks - I had the resolvers node in bower.json instead of .bowerrc.  Moving it worked just fine
